cs/benchmark/Functions.cs

```diff
@@ -25,9 +25,9 @@ public void DeleteCompletionCallback(ref Key key, Empty ctx)
         {
         }
 
-        public void CheckpointCompletionCallback(string sessionId, CommitPoint commitPoint)
+        public void CheckpointCompletionCallback(int sessionID, string sessionName, CommitPoint commitPoint)
         {
-            Debug.WriteLine("Session {0} reports persistence until {1}", sessionId, commitPoint.UntilSerialNo);
+            Debug.WriteLine("Session {0} reports persistence until {1}", sessionID, commitPoint.UntilSerialNo);
```

Comment:
Nit: Include name in debug print as well?

Reply:
done